src/Websites/Websites.Test/ScenarioTests/WebAppSlotTests.ps1

```diff
@@ -440,18 +441,27 @@ function Test-SetWebAppSlot
         Assert-Null $webApp.Identity
 		
 		# Change service plan & set properties
-		$job = Set-AzWebAppSlot -ResourceGroupName $rgname -Name $appname -Slot $slotname -AppServicePlan $planName2 -HttpsOnly $true -AsJob
```

Comment:
Any reason we are removing the HttpsOnly property from the test?

Reply:
Existing test case getting failed because of the HttpsOnly property. Since there is no dependency of HttpsOnly property on this test case,we have created a new test case.